### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD009.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#17] [General]
# https://blog.3place.cn/3place/fsd/FSD009.html

https://blog.3place.cn/3place/fsd/FSD009.html

**Comment by picklecai** [comment]
静心的人要花多年的勤劳
只有时间能促进微妙的发酵

静待花开。

**Comment by picklecai** [comment]
因为彻头彻尾的矛盾
对于智者和愚人都是同样神秘

以为随便发出一串音符就是自己听不懂的方言。

**Comment by picklecai** [comment]
知识学问
崇高的威力
全世界无人知悉
唯弗思之
始能受赠与
彼将不劳而获之。
